cosmos/providers/dbt/parser/project.py

```diff
@@ -286,21 +271,17 @@ def _handle_config_file(self, path: Path) -> None:
         """
         # parse the yml file
         config_dict = yaml.safe_load(path.read_text())
-
         # iterate over the models in the config
-        if not (config_dict and config_dict.get("models")):
+        if not config_dict.get("models"):
```

Comment:
Why did we change this line? There are circumstances when `config_dict` may be `None`, based on the `yaml.safe_load` implementation:
https://github.com/yaml/pyyaml/blob/master/lib/yaml/constructor.py#L52
https://github.com/yaml/pyyaml/blob/master/lib/yaml/__init__.py#L81